Gemfile_ar50

```diff
@@ -0,0 +1,11 @@
+source "https://rubygems.org"
+
+gemspec
+
+gem 'activeresource', git: 'git://github.com/rails/activeresource', ref: 'c3cd2b535b7c5cdc12cc6a3c1e11be6c74ffa179'
+
+gem 'minitest', "~> 5.1"
+gem 'activesupport', github: 'rails/rails'
+gem 'activemodel', github: 'rails/rails'
+gem 'arel', github: 'rails/arel'
+gem 'rails-observers', github: 'rails/rails-observers'
```

Comment:
These are here to match the [current dependencies of `activeresource`](https://github.com/rails/activeresource/blob/5c4b4a8d71e3a1ab4d1bc4fc1fbfb8eab26938aa/Gemfile#L3-L6)

I believe once Rails 5 actually ships, ActiveResource will remove the dependencies on the github versions, and then we can, too.